tsconfig.node.json

```diff
@@ -0,0 +1,12 @@
+{
+  "compilerOptions": {
+    "composite": true,
+    "module": "ESNext",
+    "moduleResolution": "Node",
+    "allowSyntheticDefaultImports": true
+  },
+  "include": [
+    "web/packages/build/vite/*.ts",
+    "web/packages/teleport/vite.config.ts"
```

Comment:
I didn't notice that before, how does it decide which `tsconfig` to pick?

Reply:
How'd you mean? This applies certain a certain tsconfig to these files, and is referenced by `tsconfig.json` in the root